src/include/processor/operator/persistent/export_db.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+#include "catalog/catalog.h"
+#include "processor/operator/sink.h"
+#include "processor/result/result_set.h"
+
+namespace kuzu {
+namespace processor {
+
+class ExportDB : public PhysicalOperator {
```

Comment:
You can declare the whole class as `final`, so you don't need the final keyword for each method.